.github/workflows/build-test-package.yml

```diff
@@ -2,370 +2,11 @@ name: Build, test, package
 
 on: [push,pull_request]
 
-env:
-  itk-git-tag: "171fb2ba33a87041f99328a2f26612ff33aa9cc8"
-  itk-wheel-tag: "v5.3rc04.post3"
-  opencl-icd-loader-git-tag: "v2021.04.29"
-  opencl-headers-git-tag: "v2021.04.29"
-
 jobs:
-  build-cxx:
-    runs-on: ${{ matrix.os }}
-    strategy:
-      max-parallel: 3
-      matrix:
-        os: [ubuntu-20.04, windows-2019, macos-11]
-        include:
-          - os: ubuntu-20.04
-            c-compiler: "gcc"
-            cxx-compiler: "g++"
-            cmake-build-type: "MinSizeRel"
-          - os: windows-2019
-            c-compiler: "cl.exe"
-            cxx-compiler: "cl.exe"
-            cmake-build-type: "Release"
-          - os: macos-11
-            c-compiler: "clang"
-            cxx-compiler: "clang++"
-            cmake-build-type: "MinSizeRel"
-
-    steps:
-    - uses: actions/checkout@v2
-
-    - name: Set up Python 3.8
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.8
-
-    - name: Install build dependencies
-      run: |
-        python -m pip install --upgrade pip
-        python -m pip install ninja
-
-    - name: Get specific version of CMake, Ninja
-      uses: lukka/get-cmake@v3.21.2
-
-    - name: Download OpenCL-SDK
-      if: matrix.os == 'macos-11'
-      run: |
-        cd ..
-        git clone --recursive https://github.com/KhronosGroup/OpenCL-SDK.git
```

Comment:
This CI is somewhat customized. I don't think we can use the standard remote action in this repo.